pages/crafts/assets/img/iedc-crafts.svg

```diff
@@ -0,0 +1,834 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="500" height="500" viewBox="0 0 500 500">
```

Comment:
![image](https://user-images.githubusercontent.com/38415384/123730819-d1446780-d84b-11eb-929b-42613aff3026.png)

Bro this SVG looks good but I'll ask the creative team to come up with a style set. We need a:
- consistent color pallet
- uniform font / typography
- and minor animation..